enterprise/api/v1/serializers.py

```diff
@@ -446,7 +446,7 @@ class EnterpriseCustomerCatalogSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.EnterpriseCustomerCatalog
         fields = (
-            'uuid', 'title', 'enterprise_customer', 'enterprise_catalog_query',
+            'uuid', 'title', 'enterprise_customer', 'enterprise_catalog_query', 'created', 'modified',
```

Comment:
It looks like some test cases might need to be updated for CI to pass, around the inclusion of `created` and `modified` in the API response.